packages/iris-grid/src/mousehandlers/IrisGridCopyCellMouseHandler.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Grid,
+  GridMouseHandler,
+  GridPoint,
+  EventHandlerResult,
+  GridMouseEvent,
+} from '@deephaven/grid';
+import { ContextActionUtils } from '@deephaven/components';
+import IrisGrid from '../IrisGrid';
+
+class IrisGridCopyCellMouseHandler extends GridMouseHandler {
+  private irisGrid: IrisGrid;
+
+  constructor(irisGrid: IrisGrid) {
+    super();
+
+    this.irisGrid = irisGrid;
+    this.cursor = null;
+  }
+
+  onClick(
+    gridPoint: GridPoint,
+    grid: Grid,
+    event: GridMouseEvent
+  ): EventHandlerResult {
+    if (
+      event.altKey &&
+      !ContextActionUtils.isModifierKeyDown(event) &&
+      !event.shiftKey
+    ) {
+      this.cursor = null;
+      if (gridPoint.columnHeaderDepth !== undefined) {
+        this.irisGrid.copyColumnHeader(gridPoint.column);
+      } else {
+        this.irisGrid.copyCell(gridPoint.column, gridPoint.row);
```

Comment:
Instead of calling `copyCell` directly, we should be calling `copyRanges` with the cell as a range. That way a notification bar will be displayed indicating a copy has occurred (or if user does not have permission to copy).
Same with the `copyColumnHeader` - it will need to go through that same flow.